docs/src/modules/java/pages/access-control.adoc

```diff
@@ -144,7 +143,6 @@ If running multiple services in local development, you may want to run with ACLs
 ```yaml
 kalix-proxy:
   image: gcr.io/kalix-public/kalix-proxy:latest
-  command: -Dconfig.resource=dev-mode.conf -Dlogback.configurationFile=logback-dev-mode.xml -Dkalix.proxy.eventing.support=google-pubsub-emulator
```

Comment:
Here and above, the fragment on how to pass `-D` is irrelevant. The goal of this two documentation section is to document how to enable the ACL and how to name the service. 

Removing just to now bloat the docs with unnecessary info. 

Of course we need some context to users can recognize that we are point to the kalix-proxy service configuration in docker-compose.